flow/activities/flowable.go

```diff
@@ -82,15 +82,15 @@ func (a *FlowableActivity) CheckConnection(
 }
 
 // SetupMetadataTables implements IFlowable.SetupMetadataTables.
-func (a *FlowableActivity) SetupMetadataTables(ctx context.Context, config *protos.Peer) error {
+func (a *FlowableActivity) SetupMetadataTables(ctx context.Context, config *protos.Peer, jobName string) error {
```

Comment:
in `flow.proto` create a new message called `SetupMetadataTablesInput` that contains the `Peer` as well as `string jobName`. This makes backwards compatible changes easier in the future.